packages/react-native-reanimated/src/platform-specific/RNRenderer.ts

```diff
@@ -1,4 +1,49 @@
+/* eslint-disable camelcase */
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck
 'use strict';
-export { default as RNRenderer } from 'react-native/Libraries/Renderer/shims/ReactNative';
+
+import { isFabric } from '../PlatformChecker';
+
+function findHostInstanceFastPath(ref: React.Component) {
+  if (ref.__internalInstanceHandle && ref.__nativeTag && ref._viewConfig) {
+    // This is a native ref to a Fabric component
+    return ref;
+  }
+  if (ref._nativeTag && ref.viewConfig) {
+    // This is a native ref to a Paper component
+    return ref;
+  }
+  return undefined;
+}
+
+function resolveFindHostInstance_DEPRECATED() {
+  if (findHostInstance_DEPRECATED !== undefined) {
+    return;
+  }
+  if (isFabric()) {
+    try {
+      findHostInstance_DEPRECATED =
+        require('react-native/Libraries/Renderer/shims/ReactFabric').findHostInstance_DEPRECATED;
+    } catch (e) {
+      findHostInstance_DEPRECATED = (_ref: unknown) => null;
+    }
```

Comment:
When will that happen? Shouldn't we throw an error indeed?

Reply:
This is the previous behavior. I wonder if we should change it 🤔

Reply:
I've just followed the previous implementation, but it might be more reasonable to throw an error instead